antarest/study/storage/rawstudy/raw_study_service.py

```
@@ -240,11 +255,13 @@ def copy(
         src_path = self.get_study_path(src_meta)
         dest_path = self.get_study_path(dest_study)
 
-        shutil.copytree(src_path, dest_path)
+        shutil.copytree(src_path, dest_path, ignore=shutil.ignore_patterns("output"))
+
+        if with_outputs:
```

IMO we should raise an Exception if `with_outputs` is False and we have a non-empty `allowed_outputs` list